qucs/qucs/Makefile.am

```diff
@@ -100,15 +104,16 @@ EXTRA_DIST = \
 	bitmaps/qucsrescodes.icns \
 	bitmaps/qucstrans.icns
 
-SUFFIXES = .h .moc.cpp .qrc.cpp
+SUFFIXES = .h .moc.cpp .qrc
 
-.qrc.cpp.cpp:
-qucs.qrc.cpp: qucs.qrc
+.qrc.cpp:
```

Comment:
what is this for? Doesn't the *Suffix rules with no recipe are also meaningless* (unlike pattern rules) from the docs apply also here?